lighthouse-viewer/test/viewer-test-pptr.js

```diff
@@ -47,14 +47,13 @@ describe('Lighthouse Viewer', function() {
     browser = await puppeteer.launch({
       headless: false,
       executablePath: process.env.CHROME_PATH,
-      args: [],
     });
     viewerPage = await browser.newPage();
     viewerPage.on('pageerror', pageError => pageErrors.push(pageError));
-    await viewerPage.goto(viewerUrl, {waitUntil: 'networkidle2'});
+    await viewerPage.goto(viewerUrl, {waitUntil: 'networkidle2', timeout: 3000});
```

Comment:
These will throw if they timeout. Do you want to catch those as errors?

Reply:
> These will throw if they timeout. Do you want to catch those as errors?

I think it's fine? It'll throw and fail in `before`, which will fail the tests.